### ISSUE TITLE
Bug Report: Strange Slowdown / Stuttering Bug On A Custom Song

### ISSUE THREAD

**Issue by Average-FNF-Modder** [FunkinCrew/Funkin#3956]
### Issue Checklist

- [x] I have properly named my issue
- [x] I have checked the Issues/Discussions pages to see if my issue has already been reported

### Platform

Itch.io (Downloadable Build) - Windows

### Browser

Microsoft Edge

### Version

0.5.3

### Description (include any images, videos, errors, or crash logs)

I'm making a custom song, so after charting it to a decent degree, I put in week format and put it in the game. But when playing the week, it appears all jittery and slow for some reason. 

Here's a link to where you can watch it happening in full: https://drive.google.com/file/d/1q8eTjR__UvLO7U75sLGzAK5dPILasWTc/view?usp=drive_link

Just to make sure it's not a fault of mine, I'll paste the metadata and week .json's below. If you want any other files, let me know.

Metadata: 
```
{
  "version": "2.2.4",
  "songName": "Breaking Point",
  "artist": "Zeroh",
  "charter": "Insert_AFM",
  "divisions": 96,
  "looped": false,
  "offsets": {
    "instrumental": 0,
    "altInstrumentals": {    },
    "vocals": {    },
    "altVocals": {    }
  },
  "playData": {
    "songVariations": [    ],
    "difficulties": [
      "easy",
      "normal",
      "hard"
    ],
    "characters": {
      "player": "pico-christmas",
      "girlfriend": "nene",
      "opponent": "darnell",
      "instrumental": "",
      "altInstrumentals": [      ],
      "opponentVocals": [
        "dad"
      ],
      "playerVocals": [
        "bf"
      ]
    },
    "stage": "limoRideErect",
    "noteStyle": "funkin",
    "ratings": {
      "normal": 0
    },
    "previewStart": 0,
    "previewEnd": 0
  },
  "generatedBy": "Friday Night Funkin' - v0.5.3",
  "timeFormat": "ms",
  "timeChanges": [
    {
      "t": 0,
      "b": 0,
      "bpm": 140,
      "n": 4,
      "d": 4,
      "bt": [
        4,
        4,
        4,
        4
      ]
    }
  ]
}
```

Week file:
```
{
"version": "1.0.0",
"name": "BREAKING POINT",
"titleAsset": "storymenu/titles/week1",
"props": [
{
"assetPath": "storymenu/props/dad",
"scale": 1.0,
"offsets": [100, 60],
"animations": [
{
"name": "idle",
"prefix": "idle0",
"frameRate": 24
}
]
},
{
"assetPath": "storymenu/props/bf",
"scale": 1.0,
"offsets": [150, 80],
"animations": [
{
"name": "idle",
"prefix": "idle0",
"frameRate": 24
},
{
"name": "confirm",
"prefix": "confirm0",
"frameRate": 24
}
]
}
],
"background": "#f02f2d",
"songs": ["breaking-point"]
}
```

Yeah idk what's going on.

### Steps to Reproduce

1. Make a custom chart in a week format
2. Sometimes it might do this? It didn't do it with a prior song I made so I don't know what's causing it

**Comment by MAZ12211** [comment]
"You need access"
Just compress the video [with this tool](https://tools.rotato.app/compress) and post it here man

**Comment by Starexify** [comment]
Try to use the fnf format for the metadata and level instead because something is off with the way it is generated, it did the same for me untill i removed some stuff (idk which but I believe its in the metadata of the song for sure)
Try my version of metadata
```json
{
  "version": "2.2.4",
  "songName": "Breaking Point",
  "artist": "Zeroh",
  "charter": "Insert_AFM",
  "looped": false,
  "playData": {
    "difficulties": [ "easy", "normal", "hard" ],
    "characters": {
      "player": "pico-christmas",
      "girlfriend": "nene",
      "opponent": "darnell",
      "instrumental": "",
      "opponentVocals": [ "dad" ],
      "playerVocals": [ "bf" ]
    },
    "stage": "limoRideErect",
    "noteStyle": "funkin",
    "ratings": { "easy": 3, "normal": 4, "hard": 5 },
    "previewStart": 0,
    "previewEnd": 0
  },
  "timeFormat": "ms",
  "timeChanges": [{ "d": 4, "n": 4, "t": 0, "bt": [4, 4, 4, 4], "bpm": 140 }],
  "generatedBy": "Friday Night Funkin' - v0.5.3"
}
```

**Comment by Average-FNF-Modder** [comment]
> "You need access" Just compress the video [with this tool](https://tools.rotato.app/compress) and post it here man

Nah, it's too long to do that without it looking like shit. I made it public. Srry about that.

**Comment by Average-FNF-Modder** [comment]
> Try to use the fnf format for the metadata and level instead because something is off with the way it is generated, it did the same for me untill i removed some stuff (idk which but I believe its in the metadata of the song for sure) Try my version of metadata
> 
> {
>   "version": "2.2.4",
>   "songName": "Breaking Point",
>   "artist": "Zeroh",
>   "charter": "Insert_AFM",
>   "looped": false,
>   "playData": {
>     "difficulties": [ "easy", "normal", "hard" ],
>     "characters": {
>       "player": "pico-christmas",
>       "girlfriend": "nene",
>       "opponent": "darnell",
>       "instrumental": "",
>       "opponentVocals": [ "dad" ],
>       "playerVocals": [ "bf" ]
>     },
>     "stage": "limoRideErect",
>     "noteStyle": "funkin",
>     "ratings": { "easy": 3, "normal": 4, "hard": 5 },
>     "previewStart": 0,
>     "previewEnd": 0
>   },
>   "timeFormat": "ms",
>   "timeChanges": [{ "d": 4, "n": 4, "t": 0, "bt": [4, 4, 4, 4], "bpm": 140 }],
>   "generatedBy": "Friday Night Funkin' - v0.5.3"
> }

I will try this when I have access, thanks. Though even if it works this is still a bug lol cause the version the game generates you should function lmao.

**Comment by Average-FNF-Modder** [comment]
> Try to use the fnf format for the metadata and level instead because something is off with the way it is generated, it did the same for me untill i removed some stuff (idk which but I believe its in the metadata of the song for sure) Try my version of metadata
> 
> {
>   "version": "2.2.4",
>   "songName": "Breaking Point",
>   "artist": "Zeroh",
>   "charter": "Insert_AFM",
>   "looped": false,
>   "playData": {
>     "difficulties": [ "easy", "normal", "hard" ],
>     "characters": {
>       "player": "pico-christmas",
>       "girlfriend": "nene",
>       "opponent": "darnell",
>       "instrumental": "",
>       "opponentVocals": [ "dad" ],
>       "playerVocals": [ "bf" ]
>     },
>     "stage": "limoRideErect",
>     "noteStyle": "funkin",
>     "ratings": { "easy": 3, "normal": 4, "hard": 5 },
>     "previewStart": 0,
>     "previewEnd": 0
>   },
>   "timeFormat": "ms",
>   "timeChanges": [{ "d": 4, "n": 4, "t": 0, "bt": [4, 4, 4, 4], "bpm": 140 }],
>   "generatedBy": "Friday Night Funkin' - v0.5.3"
> }

I tried it. It was ineffective. Thanks for trying though.

**Comment by Starexify** [comment]
What about this:
```json
{
  "version": "1.0.0",
  "name": "BREAKING POINT",
  "titleAsset": "storymenu/titles/week1",
  "visible": true,
  "background": "#f02f2d",
  "songs": ["remembered"],
  "props": [
    {
      "assetPath": "storymenu/props/dad",
      "scale": 1.0,
      "offsets": [100, 60],
      "animations": [
        {
          "name": "idle",
          "prefix": "idle0",
          "frameRate": 24
        }
      ]
    },
    {
      "assetPath": "storymenu/props/bf",
      "scale": 1.0,
      "offsets": [150, 80],
      "animations": [
        {
          "name": "idle",
          "prefix": "idle0",
          "frameRate": 24
        },
        {
          "name": "confirm",
          "prefix": "confirm0",
          "frameRate": 24
        }
      ]
    }
  ]
}
```
level/week json

if not, try to change the instrumental and vocals ogg's to og ones instead of the ones generated from fnfc (thats all i could help with because thats all i did)

**Comment by Starexify** [comment]
Alright I found what the cause could be, can you show me the contents of the folder with vocals `songs/<songname>/`?
```json
"opponentVocals": [ "dad" ],
"playerVocals": [ "bf" ]
```
Check if they're correctly: `Voices-bf` and `Voices-dad` and also match the length of the instrumental !
